sdk/java/pulumi/src/test/java/com/pulumi/automation/LocalWorkspaceTest.java

```diff
@@ -933,4 +935,18 @@ void testStateDeleteForce(@EnvVars Map<String, String> envVars) throws Exception
             }
         }
     }
+
+    static String fullyQualifiedStackName(String org, String project, String stack) {
+      return String.format("%s/%s/%s", org, project, stack);
+    }
+
+    static String getTestOrg() {
+      if (System.getenv("PULUMI_ACCESS_TOKEN") != null) {
+          if (System.getenv("PULUMI_TEST_ORG") != null) {
+              return System.getenv("PULUMI_TEST_ORG");
+          }
+          return "organization";
+      }
+      return "pulumi-test";
```

Comment:
I think these two are the wrong way around, no? If `PULUMI_ACCESS_TOKEN` is set we want to return `pulumi-test` by default, and return `organization` if it's not set at all.